src/neo/IO/Caching/DataCache.cs

```diff
@@ -43,6 +43,15 @@ public class Trackable
             }
         }
 
+        /// <summary>
+        /// Try to Add a specific key, with associated value, to the current cached dictionary.
+        /// It will not ready from Internal state.
+        /// However, if previously cached into Dictionary, request may fail.
+        /// </summary>
+        /// <param name="key">Key to be possible added.
+        /// Key will not be added if value exists cached and the modification was not a Deleted one.
+        /// </param>
+        /// <param name="value">Corresponding value to be added, in the case of sucess.</param>
         public void Add(TKey key, TValue value)
```

Comment:
Should we need to add an exception description about `ArgumentException` ?

Reply:
I did not understand, @Tommo-L.

Could you please add or explain more? I almost forgot the idea...aehauehae
That is why it is good to have these comments.

Reply:
As the `Add` method may throw `ArgumentException` exception, it will be better to add an exception comment 😂 , like the following format:

```csharp
/// ....
/// <param name="value">Corresponding value to be added, in the case of sucess.</param>
/// <exception cref="ArgumentException">comments.....</exception>  <------
 public void Add(TKey key, TValue value)
```

Reply:
Done. Thanks! Good idea.